src/applications/pact-act/containers/results/1/Page1.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+import { waitForRenderThenFocus } from '@department-of-veterans-affairs/platform-utilities/ui';
+import { ROUTES } from '../../../constants';
+import { updateResultsPage1Viewed } from '../../../actions';
+import { customizeTitle } from '../../../utilities/customize-title';
+
+const ResultsSet1Page1 = ({
+  router,
+  updateResults1Viewed,
+  viewedResultsPage1,
+}) => {
+  const H1 = 'You may be eligible for VA benefits';
+  console.log('HELLO');
```

Comment:
This console statement probably should be removed

Reply:
Done 👍